shell/client/shell.html

```diff
@@ -183,6 +176,28 @@
   </ul>
 </template>
 
+<template name="notificationItem">
+  <div class="notification-item">
+    <div class="notification-header">
+      {{#if isAppUpdates}}
+        <span>App Market has updated apps</span>
+        <span title="Ignore the below updates" class="cancel-notification">Dismiss</span>
```

Comment:
Click targets must be `<a>` or `<button>`. In this case, you want buttons, since these aren't navigational. You can style them to look like links.